module/window_swallowing.lua

```diff
@@ -11,22 +11,32 @@ local helpers = require(tostring(...):match(".*bling") .. ".helpers")
 local window_swallowing_activated = false
 
 -- you might want to add or remove applications here
-local dont_swallow_classname_list = beautiful.dont_swallow_classname_list
+local parent_filter_list = beautiful.parent_filter_list
```

Comment:
You can maintain backward compatibility to the old vars by throwing a `or beautiful.dont_swallow_classname_list or` in there, same with the other one.

Reply:
added the old vars for compatibility